client/src/components/AdvancedSearch.js

```diff
@@ -216,18 +212,15 @@ const AdvancedSearch = ({
                 }}
               >
                 <Button
-                  className={Classes.POPOVER_DISMISS}
-                  intent="danger"
```

Comment:
We can use `outline-danger` if some buttons **should** indicate a dangerous event.

Reply:
This was mixing BlueprintJS props with React Bootstrap buttons (probably copy-paste remnants). The props were meaningsless in this case.